src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ModifiedControlVariableCheck.java

```diff
@@ -73,6 +82,18 @@
  * &lt;module name="ModifiedControlVariable"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * for(int i=0;i &lt; 8;i++) {
+ *   i++; // violation, control variable modified
+ * }
+ * String args1[]={"Coding", "block"};
+ * for (String arg: args1) {
+ *   arg = arg.trim(); // violation,control variable modified
```

Comment:
add space after `,`

Reply:
not done.
please fix it and reply ''done" to this thread after you see that changes are visible in github

Reply:
![image](https://user-images.githubusercontent.com/42171790/111101345-b20d1d00-856f-11eb-97e2-8edad66d1b85.png)

I added space after "Coding" in args1[ ].@romani is that you are asking for?

Reply:
Github is not showing this one outdated

Reply:
Extremely sorry,I  added it after "Coding"

Reply:
`violation,control variable modified` ==> `violation, control variable modified`